libparsec/crates/types/src/local_manifest.rs

```diff
@@ -821,13 +830,35 @@ impl LocalFolderManifest {
         self
     }
 
+    // TODO: FINISH DOC !!!
+
+    /// Restore in the current manifest all the entries that were confined (according to
+    /// the given `prevent_sync_pattern`) in the `other` manifest.
+    ///
+    /// Note `other` is expected to be up to date with `prevent_sync_pattern` (i.e.
+    /// `apply_prevent_sync_pattern()` should have been called on it with this pattern).
+    ///
+    /// Clear the `local_confinement_points` and restore from `base.children` any
+    /// entry that was previously confined in the local children.
+    ///
+    /// This method can be seen as the opposite of `filter_remote_entries`.
+    ///
+    /// For example, considering a manifest with:
+    /// - `remote_confinement_points`: [1]
+    /// - `children`: `{"b.txt": 2}`
+    /// - `base.children`: `{"a.tmp": 1, "c.tmp": 3}`
+    ///
+    /// Then the resulting manifest would have an empty `remote_confinement points` and
+    /// `{"a.tmp": 1, "b.txt": 2}` as `children`.
     fn restore_local_confinement_points(
         self,
         other: &Self,
         prevent_sync_pattern: Option<&Regex>,
+        // TODO: why `timestamp` ? if we are only restoring confinement points then
+        //       the `updated` field shouldn't change !
```

Comment:
it's explained in the comment above: the `updated` field might change if the `other` manifest used a different filter.

Reply:
:+1: